command/var_init.go

```diff
@@ -54,7 +54,8 @@ func (c *VarInitCommand) Synopsis() string {
 
 func (c *VarInitCommand) AutocompleteFlags() complete.Flags {
 	return complete.Flags{
-		"-out": complete.PredictSet("hcl", "json"),
+		"-out":   complete.PredictSet("hcl", "json"),
+		"-quiet": complete.PredictNothing,
```

Comment:
We need to add the flag to the command help text too.

Reply:
Oops, thanks!